lua/telescope/make_entry.lua

```diff
@@ -369,11 +369,14 @@ function make_entry.gen_from_lsp_symbols(opts)
       msg = vim.trim(msg)
     else
       local filename = ""
-      opts.tail_path = get_default(opts.tail_path, true)
 
       if not opts.hide_filename then -- hide the filename entirely
         filename = entry.filename
-        if opts.tail_path then
+        if opts.relative_path then
+          -- path relative to LSP root
+          local lsp_workspaces = vim.api.nvim_buf_call(vim.fn.bufnr('#'), vim.lsp.buf.list_workspace_folders)
+          filename = vim.tbl_isempty(lsp_workspaces) and filename or utils.path_relative(filename, lsp_workspaces[1])
```

Comment:
What about setups with multiple servers attached? I think `workspace_symbols` work with multiple servers

Reply:
I thought I'd left a `--TODO` in regarding multiple servers. It'd got in my way when I was experimenting with workspace codeactions too.
I'll come back to that shortly.